packages/config/config/devices/0x031e/vzw31-sn.json

```
@@ -0,0 +1,699 @@
+					"value": 3
+				}
+			]
+		},
+		{
+			"#": "25",
+			"label": "Increase Output Power (Non-Neutral)",
+			"description": "Increase output power when in non-neutral mode. (May cause problems with high level flicker or aux switch detection. Adjust maximum level if you are having problems.)",
+			"$import": "~/templates/master_template.json#base_enable_disable"
+		},
+		{
+			"#": "50",
+			"label": "Button Delay Time",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 9,
+			"defaultValue": 5,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "No delay (config button still has 500 ms delay)",
+					"value": 0
+				},
+				{
+					"label": "300 ms",
+					"value": 3
+				},
+				{
+					"label": "400 ms",
+					"value": 4
+				},
+				{
+					"label": "500 ms",
+					"value": 5
+				},
+				{
+					"label": "600 ms",
+					"value": 6
+				},
+				{
+					"label": "700 ms",
+					"value": 7
+				},
+				{
+					"label": "800 ms",
+					"value": 8
+				},
+				{
+					"label": "900 ms",
+					"value": 9
+				}
+			]
+		},
+		{
+			"#": "52",
+			"label": "Smart Bulb Mode",
+			"$import": "~/templates/master_template.json#base_enable_disable"
+		},
+		{
+			"#": "53",
+			"$if": "firmwareVersion < 0.11",
+			"label": "Double Up to Max Level",
+			"$import": "~/templates/master_template.json#base_enable_disable"
+		},
+		{
+			"#": "54",
+			"$if": "firmwareVersion < 0.11",
+			"label": "Double Down to Min Level",
+			"$import": "~/templates/master_template.json#base_enable_disable"
+		},
+
+		{
+			"#": "53",
+			"label": "Double Up to Param 55 Level",
+			"$import": "~/templates/master_template.json#base_enable_disable"
+		},
+		{
+			"#": "54",
+			"label": "Double Down to Param 56 Level",
+			"$import": "~/templates/master_template.json#base_enable_disable"
+		},
+		{
+			"#": "55",
+			"$if": "firmwareVersion >= 0.11",
+			"label": "Double Up Level",
+			"$import": "~/templates/master_template.json#base_1-99_nounit",
+			"defaultValue": 99
+		},
+		{
+			"#": "56",
+			"$if": "firmwareVersion >= 0.11",
+			"label": "Double Down Level",
+			"$import": "~/templates/master_template.json#base_0-99_nounit"
+		},
+		{
+			"#": "58",
+			"$if": "firmwareVersion >= 0.11",
+			"label": "Exclusion Behavior",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 1,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Light does not pulse",
+					"value": 0
+				},
+				{
+					"label": "Light pulses blue",
+					"value": 1
+				},
+				{
+					"label": "Exclusion disabled (must factory reset to exclude, or change this parameter)",
+					"value": 2
+				}
+			]
+		},
+		{
+			"#": "59[0x1]",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Forward Local Commands to Associated Devices",
+			"description": "When enabled, device will send commands to associated devices when controlled locally",
```

This isn't really forwarding commands, is it? How about this?
```suggestion
			"label": "Send Local Commands To Associated Devices",
```